sparkctl/cmd/create.go

```diff
@@ -177,6 +181,29 @@ func createSparkApplication(app *v1beta2.SparkApplication, kubeClient clientset.
 
 	fmt.Printf("SparkApplication \"%s\" created\n", app.Name)
 
+	if WaitTillCompletion {
+		for {
+			appStatus, err := getSparkApplication(app.Name, crdClient)
+			if err != nil {
+				return fmt.Errorf("failed to get SparkApplication %s: %v", app.Name, err)
+			}
+
+			fmt.Printf("SparkApplication \"%s\" status: \"%s\"\n", app.Name,appStatus.Status.AppState.State)
+
+			if appStatus.Status.AppState.State == v1beta2.FailedState ||
+				appStatus.Status.AppState.State == v1beta2.UnknownState {
```

Comment:
Should this just be ignored and continue?

Reply:
What are you suggesting to do?
Do you intend to remove the check on sparkapplication in failed state or peraphs you suggest to remove the return of the error and only print an error log?

Reply:
I meant for the unknown state, we probably should ignore.

Reply:
Ok, we only add it to avoid that a unknown state of the job , probaby caused from a k8s pod error,  block the execution of the command.